src/katgpucbf/__init__.py

```diff
@@ -39,10 +39,13 @@
 # is better to minimise the number of packets/second to process.
 DEFAULT_PACKET_PAYLOAD_BYTES: Final = 8192
 DEFAULT_TTL: Final = 4  #: Default TTL for spead multicast transmission
-DEFAULT_KATCP_HOST = ""  # All interfaces
-DEFAULT_KATCP_PORT = 7147
-DIG_HEAP_SAMPLES = 4096
+DEFAULT_KATCP_HOST: Final = ""  # All interfaces
+DEFAULT_KATCP_PORT: Final = 7147
+DIG_HEAP_SAMPLES: Final = 4096
 DIG_SAMPLE_BITS: Final = 10
+# Update rate (expressed in seconds) for katcp sensors where the underlying value
+# may update extremely rapidly.
+SENSOR_UPDATE_TIME: Final = 1.0
```

Comment:
I appreciate the comment does explain it clearly enough, but the variable/constant name still doesn't sit quite right with me (rate vs `_TIME`). Bikeshedding, you might rightly say.

Reply:
What would you prefer to call it? Or is your beef with the use of "rate" in the comment?

Reply:
It's not quite an 'update **period**', is it? Because it's used as some kind of 'minimum time'? Or rather, the shortest time before an update of the sensor is sent out.

But it's likely I'm not fully understanding, you must have some reasoning as to why 'rate' is appropriate here? (No beef, just seeking clarity)

Reply:
I guess `period` would be better than `rate`, since it's in units of seconds rather than Hz. For the rx sensors it's a minimum period: if the underlying sensor updates less frequently than this then updates will be sent when the underlying sensor updates. For the time sync sensors it is just the period, since we have no visibility of the updates to the underlying kernel variables.

Reply:
@amishatishpatel See what you think of the update I just pushed.